apps/settings/src/components/AppList/AppItem.vue

```diff
@@ -111,6 +111,10 @@
 				@click.stop="forceEnable(app.id)">
 				{{ forceEnableButtonText }}
 			</NcButton>
+
+			<DaemonSelectionModal v-if="app?.app_api && showSelectDaemonModal"
+				:show.sync="showSelectDaemonModal"
```

Comment:
You use `v-if` and `show` prop, one of both should be enough, as the `show` is useless - its already removed from DOM with the `v-if`.

Maybe its a bit faster if you keep the `v-if` for the first condition - if the dialog should be shown multiple times.
```suggestion
			<DaemonSelectionModal v-if="app?.app_api"
				:show.sync="showSelectDaemonModal"
```